apps/explorer/src/components/Layout/PageLayout.tsx

```diff
@@ -26,7 +44,101 @@ export type PageLayoutProps = {
 
 const DEFAULT_HEADER_HEIGHT = 68;
 
+function ImageLink({ type }: { type: RedirectExplorer }) {
+	const { suiscanUrl, suivisionUrl } = useRedirectExplorerUrl();
+
+	const href = type === RedirectExplorer.SUISCAN ? suiscanUrl : suivisionUrl;
+	const src = type === RedirectExplorer.SUISCAN ? suiscanImg : suivisionImg;
+	const srcSet =
+		type === RedirectExplorer.SUISCAN
+			? `${suiscanImg} 1x, ${suiscanImg2x} 2x`
+			: `${suivisionImg} 1x, ${suivisionImg2x} 2x`;
+
+	return (
+		<div className="relative overflow-hidden rounded-3xl border border-gray-45 transition duration-300 ease-in-out hover:shadow-lg">
+			<ButtonOrLink href={href} target="_blank" rel="noopener noreferrer">
+				<Image src={src} srcSet={srcSet} />
+			</ButtonOrLink>
+			<div className="absolute bottom-10 left-1/2 right-0 flex -translate-x-1/2 sm:w-80">
+				<ButtonOrLink
+					className="flex w-full items-center justify-center gap-2 rounded-3xl bg-sui-dark px-3 py-2"
+					href={href}
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					<Text variant="body/semibold" color="white">
+						{type === RedirectExplorer.SUISCAN ? 'Visit Suiscan.xyz' : 'Visit Suivision.xyz'}
+					</Text>
+					<ArrowRight12 className="h-3 w-3 -rotate-45 text-white" />
+				</ButtonOrLink>
+			</div>
+		</div>
+	);
+}
+
+function RedirectContent() {
+	return (
+		<section className="flex flex-col justify-center gap-10 sm:flex-row">
+			{redirectExplorers.map((type) => (
+				<ImageLink key={type} type={type} />
+			))}
+		</section>
+	);
+}
+
+function HeaderLink({ type }: { type: RedirectExplorer }) {
+	const { suiscanUrl, suivisionUrl } = useRedirectExplorerUrl();
+	const openWithLabel =
+		type === RedirectExplorer.SUISCAN ? 'Open in Suiscan.xyz' : 'Open in Suivision.xyz';
+
+	return (
+		<ButtonOrLink
```

Comment:
We probably want an amplitude metric here

Reply:
Added amplitude